Sources/MapboxMaps/Foundation/Extensions/Core/GlyphsRasterizationOptions.swift

```diff
@@ -3,26 +3,48 @@ import MapboxCoreMaps
 
 extension GlyphsRasterizationOptions {
 
-    /// Default fallback font
-    internal static var fallbackFontFamilyName: String = {
-        UIFont.systemFont(ofSize: 0, weight: .regular).familyName
-    }()
-
-    /// Default GlyphsRasterizationOptions. RasterizationMode defaults to
-    /// `.ideographsRasterizedLocally` i.e. ideographic symbols are rasterized locally (not loaded
-    /// from the server) using an appropriate system font.
-    public static var `default`: GlyphsRasterizationOptions = {
-        GlyphsRasterizationOptions(rasterizationMode: .ideographsRasterizedLocally)
-    }()
-
     /// Convenience initializer
     /// - Parameters:
-    ///   - rasterizationMode: Rasterization mode
-    ///   - fontFamilies: Array of fonts, used for glyph rendering. Defaults to an appropriate
-    ///   system font
-    public convenience init(rasterizationMode: GlyphsRasterizationMode,
+    ///   - rasterizationMode: Rasterization mode. Defaults to
+    ///         `.ideographsRasterizedLocally` i.e. ideographic symbols are
+    ///         rasterized locally (not loaded from the server)
+    ///   - fontFamilies: Array of fonts, used for glyph rendering.Defaults to
+    ///         an appropriate system font.
+    ///
+    /// - Note:
+    ///     If `fontFamilies` is not provided, the SDK will first look for a font
+    ///     family (or array of) under the key `MBXIdeographicFontFamilyName`
+    ///     in the application's Info.plist. If one is not found, then a system
+    ///     font is returned.
+    ///
+    /// - Note:
+    ///     Calling `GlyphsRasterizationOptions()` will currently not call this
+    ///     initializer. TODO: mark underlying initializer with NS_UNAVAILABLE
+
+    public convenience init(rasterizationMode: GlyphsRasterizationMode = .ideographsRasterizedLocally,
                             fontFamilies: [String] = []) {
-        let fontFamilies = fontFamilies.isEmpty ? Self.fallbackFontFamilyName : fontFamilies.joined(separator: "\n")
+        let resolvedFamilies: [String]
+        if fontFamilies.isEmpty {
+            resolvedFamilies = Self._defaultFontFamilies()
+        } else {
+            resolvedFamilies = fontFamilies
+        }
+        let fontFamilies = resolvedFamilies.joined(separator: "\n")
         self.init(rasterizationMode: rasterizationMode, fontFamily: fontFamilies)
     }
+
+    /// Return the default font family/families
+    public static func _defaultFontFamilies() -> [String] {
```

Comment:
Can this be internal?

Reply:
Yes it can. (It used to be a default parameter)